rocketchat_API/rocketchat.py

```diff
@@ -42,14 +42,19 @@ def __reduce_kwargs(kwargs):
 
     def __call_api_get(self, method, **kwargs):
         args = self.__reduce_kwargs(kwargs)
-        return self.req.get(self.server_url + self.API_path + method + '?' +
-                            '&'.join([i + '=' + str(args[i])
-                                      for i in args]),
-                            headers=self.headers,
-                            verify=self.ssl_verify,
-                            proxies=self.proxies,
-                            timeout=self.timeout
-                            )
+        types = args.pop('types', None)
```

Comment:
I was thinking of something more generic in this case since the `types` parameter might be used for something else that might not be a list in other methods.
I believe that checking if it's a list would make more sense at this point.
What do you think?

Reply:
Agreed :).
Sorry I  did not expect such a quick review. Thanks for all your comments and advice !
I ended up fixing it in a way that works but is not very readable I think.
If you have any more tips, I'm all ears :)